coffeecard/CoffeeCard.WebApi/Startup.cs

```diff
@@ -163,6 +163,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
                 endpoints.MapRazorPages();
+                endpoints.MapFallbackToPage("/result");
```

Comment:
Does this mean that pages which does not match a routing will be routed to the result page?

Reply:
Yes, and the fallback of the result page is to give an error saying this is not the page you are looking for.